portal-api/src/v1/users/routes.js

```diff
@@ -154,32 +155,67 @@ module.exports.findById = (req, res, next) => {
 };
 
 module.exports.updateById = (req, res, next) => {
-  findOne(req.params._id, (error, user) => {
-    if (error) {
-      next(error);
-    } else if (user) {
-      const errors = applyUpdateRules(user, req.body);
-      if (errors.length) {
-        res.status(400).json({
-          success: false,
-          errors: {
-            count: errors.length,
-            errorList: combineErrors(errors),
-          },
-        });
+  if (req.user.roles.includes(ADMIN)) {
+    findOne(req.params._id, (error, user) => {
+      if (error) {
+        next(error);
+      } else if (user) {
+        const errors = applyUpdateRules(user, req.body);
+        if (errors.length) {
+          res.status(400).json({
```

Comment:
As above, it's better practice to do `return res.status(400)...` etc so that you can avoid nesting. It also ensures that any code below this point will not be executed if this statement is reached, whilst the current implementation implies that it is possible to run the code below your `res.status`